app-modules/consent/src/Filament/Resources/ConsentAgreementResource/Pages/EditConsentAgreement.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Assist\Consent\Filament\Resources\ConsentAgreementResource\Pages;
+
+use Filament\Forms\Form;
+use Filament\Forms\Components\Fieldset;
+use Filament\Forms\Components\Textarea;
+use Filament\Forms\Components\TextInput;
+use Filament\Resources\Pages\EditRecord;
+use Assist\Consent\Filament\Resources\ConsentAgreementResource;
+
+class EditConsentAgreement extends EditRecord
+{
+    protected static string $resource = ConsentAgreementResource::class;
+
+    public function form(Form $form): Form
+    {
+        return parent::form($form)
+            ->schema([
+                TextInput::make('type')
+                    ->readonly()
```

Comment:
What is the desired behaviour here? I think you might want `disabled()`, so the field is not editable via JavaScript. `readonly()` is essentially just frontend styling and does not protect the field in the backend.

Reply:
The desired behavior is disabling - updated.